ddtrace/tracer.py

```diff
@@ -445,26 +466,6 @@ def start_span(
         Note: be sure to finish all spans to avoid memory leaks and incorrect
         parenting of spans.
         """
-        is_new_process = self._check_new_process()
```

Comment:
do we still use/need `_check_new_process`?

Reply:
Ah no! I just forgot to remove it! 😢